src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -2821,7 +2845,12 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
         llvm::Value *left_val = tmp;
         this->visit_expr_wrapper(x.m_right, true);
         llvm::Value *right_val = tmp;
-        tmp = lfortran_strop(left_val, right_val, "_lfortran_strrepeat");
+        if (ASRUtils::expr_type(x.m_right)->type == ASR::ttypeType::Integer) {
+            tmp = lfortran_strrepeat(left_val, right_val);
+        }
+        else if (ASRUtils::expr_type(x.m_left)->type == ASR::ttypeType::Integer) {
+            tmp = lfortran_strrepeat(right_val, left_val);
+        }
```

Comment:
P.S. Once we create a `StringRepeat` node, we should swap the arguments in AST->ASR phase, so that the first argument is always a string, and a second argument an integer.

Reply:
Noted, I'll implement that out in another PR. I presume it should be similar to #447.

Reply:
Yes, something like that. If you don't get to it, just open up an issue please, so that we don't forget.

Reply:
Sure, here it is #462.